pkg/app/ops/insightcollector/deployments_test.go

```diff
@@ -1058,24 +1058,24 @@ func TestGroupDeployments(t *testing.T) {
 				d211,
 			},
 			apps: map[string][]*model.Deployment{
-				"application-1-1": []*model.Deployment{
```

Comment:
looks like this file not related to the other part of this PR, pls checkout this file from this PR 🙏